contrib/freeze_appimage_distro.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
```

Comment:
Should this file maybe place instead into `contrib/build-linux/appimage/`? and if so, renamed, I guess.
In particular, note that the same original issue also applies to the wine build, hence this kind of solution could be used there as well.

Reply:
I don't have a strong preference where the script resides. However maybe we can extend the script to pin, in one go, the timestamp for the appimage AND the wine build (and maybe the android build container as well), which would make contrib/ the obvious location. What do you think?

Reply:
Sounds good. If the script touches both appimage and wine, than it can be in `contrib/` directly.